File: resolucoes/python/lista01/ex007.py
# Crie um algoritmo que leia um número real e mostre na tela o seu dobro e a sua terça parte.

num = float(input('Digite um número: '))

dobro = num * 2
terca_parte = num / 3

print(f'O dobro de {num} é {dobro}')
print(f'A terça parte de {num} é {terca_parte:.3f}')

File: resolucoes/kotlin/lista01/src/main/kotlin/ex007.kt
import java.math.BigDecimal
import java.util.Scanner

/**
 * Crie um algoritmo que leia um número real e mostre na tela o seu dobro e a
 * sua terça parte.
 * Ex:

 * */
fun ex007(){
    val scanner = Scanner(System.`in`)
    print("Digite um número: ")
    val number = scanner.nextBigDecimal()
    println("""
         O dobro de $number é ${number * BigDecimal(2)}
         A terça parte de $number é ${number / BigDecimal(3)}
    """.trimIndent())
}

internal fun main(){
    ex007()
}